src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -139,6 +143,29 @@ public boolean submitHearingEvidence(String identifier, EvidenceDescription desc
                 .orElse(false);
     }
 
+    public boolean deleteDraftHearingEvidence(String identifier, String evidenceId) {
+        return deleteEvidence(identifier, evidenceId, draftHearingDocumentExtractor);
+    }
+
+    private <E> boolean deleteEvidence(String identifier, String evidenceId, DocumentExtract<E> documentExtract) {
+        return onlineHearingService.getCcdCaseByIdentifier(identifier)
+                .map(caseDetails -> {
+                    List<E> documents = documentExtract.getDocuments().apply(caseDetails.getData());
```

Comment:
what does this line do?

Reply:
Its using the functional interface to get all documents from sscscasedata. This is existing code as we have migrated cor-backend to tribunal